flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -204,6 +208,7 @@ const static struct vtol_fsm_transition fsm_land_home[FSM_STATE_NUM_STATES] = {
 	[FSM_STATE_PRE_RTH_RISE] = {
 		.entry_fn = go_enable_rise_here,
 		.static_fn = do_ph_climb,
+		.timeout = 10 * MILLI,	/* Not sure this is good */
```

Comment:
The thinking here was that if it is for some reason not getting up there, give up and start coming home. We can increase timeout.